.pre-commit-config.yaml

```diff
@@ -19,3 +19,13 @@ repos:
     rev: v1.1.0
     hooks:
     -   id: python-use-type-annotations
+-   repo: local
+    hooks:
+    -   id: flynt
+        name: flynt
+        entry: flynt
+        args: [--fail-on-change]
+        types: [python]
+        language: python
+        additional_dependencies:
+        -   flynt
```

Comment:
Does this need to be added to the dependencies files?

Reply:
Not for the pre-commit to work - it will install it automatically if needed - but perhaps it would simplify things to add flynt to dev dependencies anyway. Will do that in a bit